portfolio/src/main/webapp/script.js

```diff
@@ -336,9 +337,24 @@ async function addMarker(position) {
     displayMarkers();
 }
 
+async function checkStatus() {
+    const response = await fetch('/login/status');
+    const status = await response.text();
+    if (status.trim().localeCompare("true") === 0) {
```

Comment:
We could have this endpoint return JSON, if you want to simplify this comparison. JSON has native support for boolean values.

Reply:
I used a JSON string instead of true false string in the newest commit.